package.json

```diff
@@ -30,6 +30,7 @@
     "crypto-js": "3.1.6",
     "lodash": "3.10.1",
     "node-uuid": "1.4.7",
+    "pako": "1.0.3",
```

Comment:
for node, `request` module itself flushes out gzipped (raw response) content on `response` event. (see above).  
`pako` is only used for gzipping browser response body.